doc/deployment_guide.md

```diff
@@ -49,5422 +49,6064 @@ Red Hat®, Red Hat Enterprise Linux®, and Ceph are trademarks or registered tra
-
-> NOTE: The files are also open sourced and can be obtained from <a href="https://github.com/dsp-jetpack/JetPack/tree/JS-13.0" target="_blank">https://github.com/dsp-jetpack/JetPack/tree/JS-13.0</a>
-
-> NOTE: The automated install also requires that you have the ISO file “Red Hat Enterprise Linux 7.6 Binary DVD”. It can be downloaded from the Red Hat Customer Portal here: https://access.redhat.com/downloads/content/69/ver=/rhel---7/7.2/x86_64/product-software
-
-
-<div style="page-break-after: always;"></div>
-
-# Chapter 2 Red Hat Subscriptions
-Once all prerequisites have been met, you must determine the Appropriate Red Hat subscription entitlements for each cluster node.
-
-## Red Hat Subscription Manager Pool IDs
-You must determine the pool ID to use for the Solution Admin Host (SAH) and each node in the cluster before proceeding with the installation. To determine the pool IDs, you must have an existing server that is registered to the Red Hat Hosted Services. This server must also be registered using the same credentials as the ones being used in this environment.
-
-
-1. Once the server is correctly registered, execute the following command to see the available subscription pools.
-
-```bash
-$ subscription-manager list --all --available
-```
-
-> Note: The command will output a list of available pools. Each section of information lists what the subscription provides, its pool ID, how many are available, the type of system it is for, as well as other information.
-
-
+- [Appendix H Neutron DVR support in JS 13.2](#appendix-h-neutron-dvr-support-in-js-13.2)
+
+- [Appendix I Octavia](#appendix-i-octavia)
+
+- [Appendix J Unity with Manila and Cinder](#appendix-j-unity-with-manila-and-cinder)
+
+- [Appendix K Barbican](#appendix-k-barbican)
+
+- [Appendix L Satellite](#appendix-l-satellite)
+
+- [Appendix M Bluestore](#appendix-m-bluestore)
+
+- [Appendix N Performance optimization](#appendix-n-performance-optimization)
+
+- [Appendix O Sample files](#appendix-o-sample-files)
+
+- [Appendix P Solution validation overview](#appendix-p-solution-validation-overview)
+
+- [Appendix Q References](#appendix-q-references)
+
+
+
+
+<div style="page-break-after: always;"></div>
+
+# Chapter 1 Overview
+This guide provides information necessary to deploy the Dell EMC Ready Architecture for Red Hat OpenStack platform v13.2 using an automation framework developed by Dell EMC.
+
+### General Hardware Options
+
+To reduce time spent on specifying hardware for an initial system, the Architecture Guide offers a full solution using validated Dell EMC PowerEdge server hardware designed to allow a wide range of configuration options, including optimized configurations for: 
+
+* Compute nodes
+* Infrastructure nodes
+* Storage nodes
+
+Dell EMC recommends starting with OpenStack software using components from this Architecture Guide -
+Version 13.2 flexible foundation expands as your cloud deployment grows, which protects investments.
```

Comment:
This whole paragraph needs to be re-written.  It doesn't flow at all, it isn't an Architecture Guide and it doesn't make sense.